qml/pages/about/AboutBerlinVegan.qml

```diff
@@ -222,6 +226,31 @@ BVApp.Page {
                 }
             }
 
+            Item {
+                width: parent.width
+                height: BVApp.Theme.paddingLarge
+            }
+
+            Label {
+                         //% "Dutch translations:"
```

Comment:
Why did you choose plural here? Normally in German and English singular is used in that case:
"Dutch translation" / "Niederländisch Übersetzung:"